### PR TITLE
Bump up deployment version in tasks/access-application-cluster folder

### PR DESCRIPTION
**Pull request by nashasha1**
I will create a bunch of PRs for bumping up deployment version.  A pr contains all changes in a folder.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/7324)

### CODE REVIEW

**Comment by steveperry-53** [comment]
@xiangpengzhao Do you still want to wait for a review from @janetkuo? Or do you think it would be OK to merge this?

**Comment by nashasha1** [comment]
ping @xiangpengzhao

File: docs/tasks/access-application-cluster/frontend.yaml [hunk 1]
```diff
@@ -12,11 +12,15 @@ spec:
...
-apiVersion: apps/v1beta1
+apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: frontend
 spec:
+  selector:
+    matchLabels:
+      app: hello
+      tier: frontend
```
Comment (janetkuo) [inline docs/tasks/access-application-cluster/frontend.yaml]: Need `track: stable` too

File: docs/tasks/access-application-cluster/hello.yaml [hunk 2]
```diff
@@ -1,8 +1,12 @@
-apiVersion: apps/v1beta1
+apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: hello
 spec:
+  selector:
+    matchLabels:
+      app: hello
+      tier: backend
```
Comment (janetkuo) [inline docs/tasks/access-application-cluster/hello.yaml]: `track: stable`

**Comment by xiangpengzhao** [comment]
yeah, @janetkuo 's review is definitely necessary I think :)

File: docs/tasks/access-application-cluster/hello.yaml (hunk 2 above)
Comment (nashasha1) [reply docs/tasks/access-application-cluster/hello.yaml]: @janetkuo  Done!

**Comment by nashasha1** [comment]
@xiangpengzhao  @janetkuo  Thanks for your reply.

**Comment by xiangpengzhao** [comment]
/lgtm
after @janetkuo comment addressed.

**Pull request merged**